video_worker/tests/test_video_worker.py

```diff
@@ -162,7 +151,7 @@ def test_run(self, mock_data, mock_mkdir, mock_exists, mock_rmtree, mock_logger,
 
         # First 2 calls to os.path.exists will be called in WS.run() so we need to make sure our TEST_INSTANCE_YAML
         # file path is check correctly.
-        mock_exists.side_effect = [True, True, path_exists]
+        mock_exists.side_effect = mock_data.get('exists_side_effects', [True, True, True])
```

Comment:
We don't need `path_exists = mock_data.get('path_exists', True)` at L#43 anymore.